### PR TITLE
[DOC] Add changes for upcoming version

### PR DESCRIPTION
**Pull request by InvisibleKind**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,10 +1,30 @@
 # Fluidcontent Change log
 
+upcoming
+------------------
+
+- :exclamation: Support of [TYPO3 6.2 dropped](https://github.com/FluidTYPO3/fluidcontent/commit/f1df9a4b7ee563335c8f1feeb7925e6f820a7f19)
```
Comment (NamelessCoder) [inline CHANGELOG.md]: Let's use this space to refer to the new `legacy` branches which users can use for 6.2 ;)

Comment (InvisibleKind) [inline CHANGELOG.md]: @NamelessCoder you mean, there should be now 2 **upcoming** sections: first - for 6.2 and changelog for legacy branches should be there, second section - for 7.high_five_shark version with own changelog, right?

Comment (NamelessCoder) [inline CHANGELOG.md]: Nono, just that this line where we say we drop 6.2 should also say that if you for some reason need to continue using 6.2, there is a legacy branch available. We don't want to maintain a changelog for that branch.

Comment (InvisibleKind) [inline CHANGELOG.md]: @NamelessCoder, got it! And will this branch reused https://github.com/FluidTYPO3/fluidcontent/tree/legacy or a new one created?

Comment (NamelessCoder) [inline CHANGELOG.md]: We will use the same name but I'm not sure yet if we will reuse or recreate it. The link will be the same in the end.

**Pull request merged**